docs/confounding-variables.md

```diff
@@ -0,0 +1,40 @@
+# Confounding Variables
+
+A confounding variable is a factor that influences independent variable and dependant variable. 
+A confounding variable can mix with the effect of an independent variable on the dependent variable thus leading to erronous conclusions about the experiment.
+
+At any time during your experiment, the data manager allows you to capture confounding variables. 
+
+### Define confounding variables in your experiment
+
+In order to work with confounding variables, first you must define them in your experiment. 
+Confounding variables are experiment specific. Confounding variables do not interact with
+information outside your experiment.
+
+To define a new confounding variable, navigate to your experiment, open the _Confounding Variables_ tab, click on **Add confounding variables**. In case confounding variables are defined already, click **Add** in the top right corner of the tab.
+
+After clicking the button, a dialog opens where you can **enter the names of your confounding variables**. Confirm your changes and the confounding variables are now defined in your experiment.
```

Comment:
Better without UI detail: after confirmation, provide the names for ...

Reply:
Confirmation of what? We need to tell them where to enter it. Otherwise what is the point of the documentation?